src/main/groovy/wooga/gradle/wdk/unity/WdkUnityPlugin.groovy

```diff
@@ -47,24 +57,30 @@ import wooga.gradle.wdk.unity.actions.IOSResourceCopyAction
 import wooga.gradle.wdk.unity.actions.WebGLResourceCopyAction
 
 import javax.inject.Inject
+import java.security.cert.Extension
 import java.util.concurrent.Callable
 
 class WdkUnityPlugin implements Plugin<Project> {
 
     static Logger logger = Logging.getLogger(WdkUnityPlugin)
 
+    static String GROUP = "wdk"
+    static String EXTENSION_NAME = "wdk"
+
     static String ASSEMBLE_RESOURCES_TASK_NAME = "assembleResources"
     static String ASSEMBLE_IOS_RESOURCES_TASK_NAME = "assembleIOSResources"
     static String ASSEMBLE_ANDROID_RESOURCES_TASK_NAME = "assembleAndroidResources"
     static String ASSEMBLE_WEBGL_RESOURCES_TASK_NAME = "assembleWebGLResources"
     static String SETUP_TASK_NAME = "setup"
-    static String GROUP = "wdk"
-    static String EXTENSION_NAME = "wdk"
+
     static String ANDROID_RESOURCES_CONFIGURATION_NAME = "android"
     static String IOS_RESOURCES_CONFIGURATION_NAME = "ios"
     static String WEBGL_RESOURCES_CONFIGURATION_NAME = "webgl"
     static String RUNTIME_CONFIGURATION_NAME = "runtime"
 
+    static String GENERATE_UPM_PACKAGE_TASK_NAME = "upmPack"
+    static String GENERATE_META_FILES_TASK_NAME = "generateMetaFiles"
```

Comment:
I would not yet put this as a public task to be fetchable via constants.

Reply:
I will make it private then.